conf/application.conf

```diff
@@ -30,7 +30,6 @@ play.http.errorHandler = "handlers.ErrorHandler"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
-play.filters.enabled += "uk.gov.hmrc.play.bootstrap.frontend.filters.SessionIdFilter"
```

Comment:
I haven't seen this deletion in any other Play 3 upgrades, so how come it's able to be removed here?

Reply:
It is already included by frontend.conf. See [here ](https://github.com/hmrc/bootstrap-play/blob/a31da3432213a0572af4b5613afa29db08ce248b/bootstrap-frontend-play-30/src/main/resources/frontend.conf#L134)